test/e2e/framework/volume/fixtures.go

```diff
@@ -128,12 +128,8 @@ type TestConfig struct {
 	// Wait for the pod to terminate successfully
 	// False indicates that the pod is long running
 	WaitForCompletion bool
-	// ServerNodeName is the spec.nodeName to run server pod on.  Default is any node.
-	ServerNodeName string
```

Comment:
Do we need to still have a seperate `ServerNodeSelection`? suspicious that this disappeared